draft-linker-adem-wg-adem-core.md

```diff
@@ -168,6 +177,8 @@ IPv6 addresses MUST be global unicast or link-local unicast addresses.
 
 #### Semantics
 
+//TODO(djackson) - My gut feeling would be to walk this back and focus purely on networks in the beginning. Better to solve one thing well than two things poorly? Just protect networks and their communications? 
```

Comment:
This may be a bit confusingly put. What I tried to define is: what does an emblem including an IP address actually protect? We focus solely on networks by identifying everything via a network address, but the question remains: what does a network address actually identify? Is it just the process listening on that port? Or all such processes if there is no port specified? Or is it actually all such processes and the data on the computer controlling that IP address?